jaxrs-api/src/main/java/se/fortnox/reactivewizard/jaxrs/Stream.java

```diff
@@ -11,5 +11,18 @@
 @Target({ElementType.METHOD})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Stream {
+    /**
+     * Stream format. If it set to CONCATENATED_JSON_OBJECTS,
```

Comment:
The reader may not know what format you are referring to when you write _"if it set to CONCATENATED_JSON_OBJECTS,  then the format will be just like an Observable stream"_

maybe more valuable if you state how the format will be.